pkgs/tools/audio/yabridge/default.nix

```diff
@@ -80,15 +80,10 @@ in stdenv.mkDerivation rec {
   )'';
 
   patches = [
-    # Fix printing wine version when using absolute path (remove patches in next release):
-    (fetchpatch {
-      url = "https://github.com/robbert-vdh/yabridge/commit/2aadf5256b3eafeb86efa8626247972dd33baa13.patch";
-      sha256 = "sha256-Nq9TQJxa22vJLmf+USyPBkF8cKyEzb1Lp2Rx86pDxnY=";
-    })
-    (fetchpatch {
-      url = "https://github.com/robbert-vdh/yabridge/commit/93df3fa1da6ffcc69a5b384ba04e3da7c5ef23ef.patch";
-      sha256 = "sha256-//8Dxolqe6n+aFo4yVnnMR9kSq/iEFE0qZPvcIBehvI=";
-    })
+    # Fix for wine 6.8+ (remove patch in next release):
+    # Patch is not directly fetched from upstream, because it causes a conflict in the CHANGELOG.
+    # This copy has that changes removed.
+    ./wine-6.8.patch
```

Comment:
You can use something like
```nix
fetchpatch {
  # ...
  excludes = [ "CHANGELOG" ];
}
```

Reply:
Oh, neat. Fixed.
Is this feature documented somewhere explicitly or is this something you 'just need to know' / inspect the source for?

Reply:
> Is this feature documented somewhere explicitly or is this something you 'just need to know' / inspect the source for?

I don't think it is. But if you are feeling like it, the right place would be https://github.com/NixOS/nixpkgs/blob/master/doc/builders/fetchers.chapter.md

Reply:
@SFrijters  Also note, that 6.9 staging almost brownbag (dx11 games don't  work at all).
```
   staging = fetchFromGitHub rec {
     # https://github.com/wine-staging/wine-staging/releases
     inherit (unstable) version;
-    sha256 = "sha256-hLst6vrg8qFiT7QM9nOQ9dziaVLyUw0I6In8E9waod0=";
+    sha256 = "sha256-g0NmiypafOAmKDRoRf4uz5NnhFo6uga0fKYNCF29jbE=";
     owner = "wine-staging";
     repo = "wine-staging";
-    rev = "v${version}";
+    #rev = "v${version}";
+    # FIXME: replace with line above with 6.10 release
+    # Fix https://bugs.winehq.org/show_bug.cgi?id=51172
+    rev = "5bbe3e47a559b3c04bc8791e0b398a271c772af7"; 
 
     # Just keep list empty, if current release haven't broken patchsets
-    disabledPatchsets = [ ];
+    # Actually only "d3d11-Deferred_Context" cause problems, two others only dependencies
+    # see FIXME above
+    disabledPatchsets = [ "d3d11-Deferred_Context" "wined3d-CSMT_Main" "nvapi-Stub_DLL" "nvcuvid-CUDA_Video_Support" "nvencodeapi-Video_Encoder" ];
   };
```
(next commit after tag on staging, +some disabling, because I can't add third commit there, but skip second one.
probably we need some postStagingPatches as well)

Reply:
Maybe not worth to do it now then? I could remove 6.8 -> 6.9 altogether and see what 6.10 brings?